65black/StackOlite
# #160201422 Post An Answer

#### What does this PR do?
This PR consumes the `/api/v1/questions/:questionId/answers` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Log in to an existing account.
 - Navigate to questions page.
 - Open any question and add an answer.
 - The page will refresh and you should see your newly added answer at the bottom.

#### Screenshots
![answer](https://user-images.githubusercontent.com/36507972/45916815-d63a8800-be63-11e8-8bf8-6cad3646f493.JPG)


#### What are the relevant pivotal tracker stories?
#160201422
